com.google.Chrome.json

```diff
@@ -3,26 +3,18 @@
     "runtime": "org.freedesktop.Platform",
     "runtime-version": "1.6",
     "sdk": "org.freedesktop.Sdk",
-    "command": "/app/bin/eos-google-chrome-app",
```

Comment:
The launcher in the OS depends on this entry point existing, so I don't think we should change it (or we provide a compat symlink) - https://github.com/endlessm/eos-browser-tools/blob/master/chrome-helper/eos-google-chrome.py#L86

This means the script needs to handle two things - either running inside the flatpak and doing the spawn trick, or outside and maybe just re-execing itself with flatpak, for consistency (and no need to do the pipe that keeps the flatpak alive etc).

To rely on the regular launch behaviour in the OS, we can use an eufi "upgrade" method in the OS so that we can drop the launcher hacks, but we can't sequence the upgrade the other way (prevent an older OS getting a new Flatpak).

Reply:
My idea for the transition was that the "self-contained" version of the app was only ever hosted on Flathub, not on eos-apps. (In that sense this PR is more of a RFC than something to merge right away here.)
So older OS would never get this new one and we would not worry about keeping the legacy launcher; in the same OS update we would remove all the launcher stuff and add the migration to the Flathub version.
Is that possible?